gdshelpers/parts/waveguide.py

```diff
@@ -409,7 +409,7 @@ def add_route_single_circle_to(self, final_coordinates, final_angle, max_bend_st
         d = abs(radius * np.tan(diff_angle / 2))
 
         tmp_wg = Waveguide.make_at_port(self._current_port)
-        tmp_wg.add_straight_segment(distance[0] - d)
+        tmp_wg.add_straight_segment(length=distance[0] - d)
```

Comment:
what does the `length=` help here? as it is anywhere else used without, we should stay consistent I'd guess

Reply:
This was by mistake.